docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -56,18 +56,20 @@ The deployer account is the account that will deploy your contracts. Additionall
 
 You can generate a random account / private key or add your crypto wallet's private key.
 
-To create a random account and add the `DEPLOYER_PRIVATE_KEY` to the `.env` file, run:
+To create a random account and add the `DEPLOYER_PRIVATE_KEY_ENCRYPTED` to the `.env` file, run:
 
 ```
 yarn generate
 ```
 
+You will be prompted to enter a password which will be used to encrypt your private key. Make sure to remember this password as you'll need it for future deployments and account queries.
```

Comment:
If we remove https://github.com/scaffold-eth/se-2-docs/pull/104/files#r1879382950 maybe we should mention here a bit that : 

Running this command will automatically add encrypted private key (`DEPLOYER_PRIVATE_KEY_ENCRYPTED`) in `.env` file.